backend/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "c19-backend"
+  }
+}
```

Comment:
Is it possible to call this `.firebaserc.json` or `firebaserc.json`? This allows Prettier to auto-format it.